src/test/jtx/impl/JSONRPCClient.cpp

```diff
@@ -140,10 +140,57 @@ class JSONRPCClient : public AbstractClient
         Json::Reader jr;
         Json::Value jv;
         jr.parse(buffer_string(res.body.data()), jv);
-        if(jv["result"].isMember("error"))
-            jv["error"] = jv["result"]["error"];
-        if(jv["result"].isMember("status"))
-            jv["status"] = jv["result"]["status"];
+        if(jv[jss::result].isMember(jss::error))
+            jv[jss::error] = jv[jss::result][jss::error];
+        if(jv[jss::result].isMember(jss::status))
+            jv[jss::status] = jv[jss::result][jss::status];
+        return jv;
+    }
+
+    Json::Value
+    invoke(Json::Value const& cmd) override
```

Comment:
The caller must handle that.  The first `invoke` builds a 2.0 Json request object if asked to (`rpc_version_ == 2`).  This one dumps all the handholding expects a 2.0 Json request object.